app/javascript/sound-engines/sampler-engine.js

```diff
@@ -1,13 +1,25 @@
+import InstrumentService from "../services/instrument.service";
+
 class SamplerEngine {
   constructor() {
     this.name = 'Sampler';
     this.instrument = '-';
+    InstrumentService.loadSample('assets/sample.wav')
+      .then(sample => {
+          this.synthService = new InstrumentService(sample)
+      })
+
   }
 
   onPadPressed(padIndex) {
     console.log(`Pad ${padIndex} pushed from the Sampler engine`);
 
     // Implement here the logic to produce Sound based on the Pad Index (int)
+    // TODO: Play in correct synth mode only
+    if (this.synthService) {
+        this.synthService.play(padIndex / 15)
+    }
+
   }
 
   changeInstrument() {
```

Comment:
Can you install `prettier` and format the code? :) 

If needed, you can add a prettier config file, like this one:

```js
// prettier.config.js or .prettierrc.js
module.exports = {
  trailingComma: "es5",
  tabWidth: 2,
  semi: true,
  singleQuote: true,
};
````

(that's the main principle following Nimble's conventions)

Reply:
Updated at [81eb0e6](https://github.com/malparty/nimbleat/pull/16/commits/81eb0e68581f202da3f65ca436380cc260e64416)